.coveragerc

```diff
@@ -65,6 +65,7 @@ omit =
     homeassistant/components/azure_event_hub/*
     homeassistant/components/azure_service_bus/*
     homeassistant/components/baidu/tts.py
+    homeassistant/components/becker/*
```

Comment:
The configuration flow should not be excluded from tests, by using * it is excluded.